src/NearestNeighbors.jl

```diff
@@ -161,7 +161,7 @@ metadata_pkg.((KNNRegressor, KNNClassifier),
 
 metadata_model(KNNRegressor,
     input   = Table(Continuous),
-    target  = Union{AbstractVector{Continuous}, AbstractVector{<:AbstractVector{Continuous}}},
+    target  = Union{AbstractVector{Continuous}, AbstractVector{<:AbstractArray{Continuous}}},
```

Comment:
Need `Table` here as in https://github.com/alan-turing-institute/MLJMultivariateStatsInterface.jl/blob/0b996da48eeb111c95dcc7e1a375a646c4e1d900/src/models/linear_models.jl#L158